<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thomas Lété</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PySSH Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressources/ressources.qrc">
    <normaloff>:/img/Activity-monitor-32.png</normaloff>:/img/Activity-monitor-32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="lwServers">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>DB Server</string>
         </property>
         <property name="statusTip">
          <string>Ok</string>
         </property>
         <property name="icon">
          <iconset resource="../../ressources/ressources.qrc">
           <normaloff>:/img/Tick-32.png</normaloff>:/img/Tick-32.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NAS</string>
         </property>
         <property name="icon">
          <iconset resource="../../ressources/ressources.qrc">
           <normaloff>:/img/Warning-32.png</normaloff>:/img/Warning-32.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HTTP Server 1</string>
         </property>
         <property name="icon">
          <iconset resource="../../ressources/ressources.qrc">
           <normaloff>:/img/Cross-32.png</normaloff>
           <disabledoff>:/img/Cross-32.png</disabledoff>:/img/Cross-32.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pbAddServer">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../ressources/ressources.qrc">
            <normaloff>:/img/Plus-32.png</normaloff>:/img/Plus-32.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbDelServer">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../ressources/ressources.qrc">
            <normaloff>:/img/Cross-32.png</normaloff>:/img/Cross-32.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbEditServer">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../ressources/ressources.qrc">
            <normaloff>:/img/Pencil-32.png</normaloff>:/img/Pencil-32.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="twMonitor">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset resource="../../ressources/ressources.qrc">
         <normaloff>:/img/Statistics_32.png</normaloff>:/img/Statistics_32.png</iconset>
       </attribute>
       <attribute name="title">
        <string>DB Server</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="pbOpenShell">
            <property name="text">
             <string>Open an interactive shell</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pbRefresh">
            <property name="text">
             <string>Refresh Monitoring</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pbSendCommand">
            <property name="text">
             <string>Send a command ...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>CPU Load :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Memory Occupation :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QProgressBar" name="pbMemory">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="maximum">
             <number>2048</number>
            </property>
            <property name="value">
             <number>648</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QProgressBar" name="pbCpuLoad">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="value">
             <number>15</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Running Processes :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="teProcesses">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset resource="../../ressources/ressources.qrc">
         <normaloff>:/img/Statistics_32.png</normaloff>:/img/Statistics_32.png</iconset>
       </attribute>
       <attribute name="title">
        <string>NAS</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QProgressBar" name="pbMonitorLoad">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReduce"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuServers">
    <property name="title">
     <string>Servers</string>
    </property>
    <addaction name="actionAdd_server"/>
    <addaction name="actionManage_logins_and_passwords"/>
    <addaction name="actionSend_a_command_to_server"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionRead_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuServers"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAdd_server">
   <property name="text">
    <string>Add Server</string>
   </property>
  </action>
  <action name="actionManage_logins_and_passwords">
   <property name="text">
    <string>Manage logins and passwords</string>
   </property>
  </action>
  <action name="actionReduce">
   <property name="text">
    <string>Reduce to system tray</string>
   </property>
  </action>
  <action name="actionRead_Help">
   <property name="text">
    <string>Read Help</string>
   </property>
  </action>
  <action name="actionSend_a_command_to_server">
   <property name="text">
    <string>Send a command to server</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../ressources/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
